### DISCUSSION TITLE
Why is monitorElasticCache part of default monitoring when using monitorScope?

### DISCUSSION THREAD

**Discussion by larskinder** [cdklabs/cdk-monitoring-constructs#446] [Q&A] [answered]
Hi,

I am fairly new to using the monitoring construct and think it is really good. :- ) 
What I don't get is (if I understand the TS code correctly), why `monitorElasticCache` is part of the default setup when using `monitorScope`? 
I noticed this, when taking a look at the default dashboard in CloudWatch and was like, why are their metrics for EC, if I am not using it and therefore not having it in my scope. 

Again, maybe I am misunderstanding the intention or code, but to me this seems really misplaced.

**Comment by ghost** [comment, marked as answer]
Unfortunately the implementation of `ElastiCacheClusterMonitoring`/`ElastiCacheClusterMetricFactory` relied on a plain `clusterId` to identity the clusters rather than any concrete construct nodes. That may be subject to change if L2 constructs for ElastiCache are ever made available.
